app-text/texlive/texlive-2021.recipe

```diff
@@ -384,77 +464,112 @@ PROVIDES="
 	cmd:vpl2ovp
 	cmd:vpl2vpl
 	cmd:vptovf
+	cmd:webquiz
 	cmd:weave
 	cmd:wofm2opl
 	cmd:wopl2ofm
+	cmd:wordcount
 	cmd:wovf2ovp
 	cmd:wovp2ovf
 	cmd:xdvipdfmx
 	cmd:xelatex
-	cmd:xelollipop
+	cmd:xelatex_dev
 	cmd:xetex
+	cmd:xhlatex
+	cmd:xindex
+	cmd:xml2pmx
 	cmd:xmltex
-	lib:libkpathsea$secondaryArchSuffix
-	lib:libptexenc$secondaryArchSuffix
+	cmd:yplan
+	lib:libkpathsea$secondaryArchSuffix = $libkpathseaVersionCompat
+	lib:libptexenc$secondaryArchSuffix = $libptexencVersionCompat
+	lib:libsynctex$secondaryArchSuffix = $libsynctexVersionCompat
+	lib:libtexlua53$secondaryArchSuffix = $libtexluaVersionCompat
+	lib:libtexluajit$secondaryArchSuffix = $libtexluajitVersionCompat
 	"
 REQUIRES="
 	haiku$secondaryArchSuffix
-	lib:libpng$secondaryArchSuffix
-	lib:libz$secondaryArchSuffix
-	lib:libbz2$secondaryArchSuffix
-	lib:libfreetype$secondaryArchSuffix
+	lib:libTECkit$secondaryArchSuffix
+	lib:libcairo$secondaryArchSuffix
 	lib:libfontconfig$secondaryArchSuffix
-	lib:libicudata$secondaryArchSuffix >= 66
-	lib:libicuuc$secondaryArchSuffix >= 66
-	lib:libicui18n$secondaryArchSuffix >= 66
+	lib:libfreetype$secondaryArchSuffix
+	lib:libgd$secondaryArchSuffix
+	lib:libgmp$secondaryArchSuffix
+	lib:libgobject_2.0$secondaryArchSuffix
+	lib:libgraphite2$secondaryArchSuffix
+	lib:libharfbuzz$secondaryArchSuffix
+	lib:libicudata$secondaryArchSuffix
+	lib:libicui18n$secondaryArchSuffix
+	lib:libicuio$secondaryArchSuffix
+	lib:libicuuc$secondaryArchSuffix
+	lib:libmpfr$secondaryArchSuffix
+	lib:libpaper$secondaryArchSuffix
+	lib:libpixman_1$secondaryArchSuffix
+	lib:libpng16$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	lib:libzzip_0$secondaryArchSuffix
 	"
 
 PROVIDES_devel="
 	texlive${secondaryArchSuffix}_devel = $portVersion
-	devel:libkpathsea$secondaryArchSuffix = $portVersion
-	devel:libptexenc$secondaryArchSuffix = $portVersion
+	devel:libkpathsea$secondaryArchSuffix = $libkpathseaVersionCompat
+	devel:libptexenc$secondaryArchSuffix = $libptexencVersionCompat
+	devel:libsynctex$secondaryArchSuffix = $libsynctexVersionCompat
+	devel:libtexlua53$secondaryArchSuffix = $libtexluaVersionCompat
+	devel:libtexluajit$secondaryArchSuffix = $libtexluajitVersionCompat
 	"
 REQUIRES_devel="
 	texlive$secondaryArchSuffix == $portVersion base
 	"
 
+ARCHITECTURES_texmf="any"
+PROVIDES_texmf="
+	texlive${secondaryArchSuffix}_texmf = $portVersion
+	"
+REQUIRES_texmf="
+	texlive$secondaryArchSuffix == $portVersion base
+	"
+
+ARCHITECTURES_texmf_doc="any"
+PROVIDES_texmf_doc="
+	texlive${secondaryArchSuffix}_texmf = $portVersion
```

Comment:
I 'think' this should be _texmf_doc (got an error for a not matching provides earlier), doing a new check atm :)

Reply:
Maybe it should even be `texlive_texmf_doc` (without `$secondaryArchSuffix` as this is a `any` package)?

Reply:
You are right, this is a copy-paste bug. Why didn't that show up with my test builds? (I used `haikuporter -S`). Is the resulting package actually called `texlive_texmf_doc` on 32 bit despite being a subpackage of `texlive_x86`? (and the same for `texlive_texmf` above)

Reply:
And if it is the same package for all architectures, can I even put a `REQUIRES=texlive$secondaryArchSuffix` there? This would break for at least one architecture then because `$secondaryArchSuffix` expands to different values.

Reply:
That's the next check ;) For now let's fix this one :)